maintainers/maintainer-list.nix

```diff
@@ -11890,6 +11890,11 @@
     github = "Mephistophiles";
     githubId = 4850908;
   };
+  mevatron = {
+    name = "mevatron";
+    github = "mevatron";
+    githubId = 714585;
+  };
```

Comment:
An email, please.

Reply:
> An email, please.
email is not required.
https://github.com/NixOS/nixpkgs/blob/0c9dd8ccc86abefc5869a175480bdecbd4588cd1/maintainers/maintainer-list.nix#L4-L12

Reply:
This is not required.
But it is still useful, so it is not wrong to ask.